templates/types/streaming/fastapi/app/engine/generate.py

```diff
@@ -67,6 +67,7 @@ def generate_datasource():
     # Set the metadata for the documents
     for doc in documents:
         doc.metadata["private"] = "false"
```

Comment:
can't we remove this?

Reply:
If we stick with nin filter then removing this is fine